tests/test_array.py

```diff
@@ -417,3 +417,193 @@ def test_update_attrs(zarr_format: int) -> None:
 
     arr2 = zarr.open_array(store=store, zarr_format=zarr_format)
     assert arr2.attrs["foo"] == "bar"
+
+
+@pytest.mark.parametrize("store", ["memory"], indirect=True)
+@pytest.mark.parametrize("zarr_format", [2, 3])
+def test_resize_1d(store: MemoryStore, zarr_format: int) -> None:
```

Comment:
to fight test bloat we should find a way to parameterize over dimensionality instead of making separate 1d, 2d, etc tests. but that's not a blocker here.

Reply:
agree! These tests were copied over from v2 so I'm going to leave them for now.